interfaces/rail/README.md

```diff
@@ -0,0 +1,118 @@
+
+### creation
+
+- **simulateWithSEDs.py**: In **TUTORIAL MODE** it generates mock dataset inside Delight. These data are the traning and validation samples of fluxes-fluxeserrors-redshift drawing ramdomly the type of the underlying SED template Fluxes-Redshifts model build in **processSEDs.py**. These data are written in the ascii files **'trainingFile'** and **'targetFile'** (which true name is given in the configuration parameter file).
+
+### Learn
+
+- **delightLearn.py**:	Build the Gaussian Process from the training data. The Gaussian process is written in the file **'training_paramFile'**.
+- whether one is running in **TUTORIAL MODE** and **STANDARD MODE**, all the training data are processed in one step.
+
+- Note for template estimation of redshift, there is no learning phase. 
+
+### Estimation	
+
+
+							
+- **templateFitting.py**: The Gaussian Process estimation is also based on latent SED templates. These lated SED template are only usefull for validation dataset to help the GP to estimate the redshift. That is why this module is called from RAIL estimation function (note Learn). Namely it writes the file **'redshiftpdfFileTemp'**.
+
+
+- **delightApply.py**:	Estimate the redshift posterior distribution using Gaussian process and writes its results in file  **'globalPDFs'**. 						
+
+## Other Delight modules
+
+- **calibrateTemplateMixturePriors.py**: This module must be used outside RAIL. It is only tool to check. 
+
+
+## Python modules for interface purposes
+
+
+- **makeConfigParam.py**: it generates Delight configuration file **parametersTest.cfg** from Delight configuration file **delightPZ.yaml**. In the case of RAIL estimation, the validation dataset is split into different chunks. In the later case, there must be is one Delight configuration file per chunk. A Delight configuration file must contain all datafile paths used by Delight.
+
+- **convertDESCcat.py**: Convert DESC PhotoZ dataset into ascii files readable by Delight. For now, only files with hdf5 format are used.
+
+- **getDelightRedshiftEstimation.py**: Retrieve results on PhotoZ estimation of Delight for RAIL. 
+
+
+## Metrics
+- Delight builds its own metrics in **templateFitting.py** and **delightApply.py**. Those modules write their metric files results in **'metricsFileTemp'** and **'globalMetrics'** respectively.
+- Because for DC2, the validation datasetp is split in chunks, it is better to use upward RAIL metrics.
+
+## Utilities
+
+- **utils.py** : Library of IO inside RAIL used in **convertDESCcat.py** for reading DESC dataset. (It means it has been copied from RAIL to Delight bacause it is not possible to make cross dependence between these packages (Delight depending on RAIL and RAIL depending on Delight).
+
```

Comment:
right but copying the codebase is not a solution either. Like in the comment above, I think that the code relying on it needs to live in rail, not in Delight